tests/models/test_gemini.py

```diff
@@ -70,12 +73,13 @@ def test_api_key_empty(env: TestEnv):
         GeminiModel('gemini-1.5-flash')
 
 
-def test_agent_model_simple(allow_model_requests: None):
+async def test_agent_model_simple(allow_model_requests: None):
     m = GeminiModel('gemini-1.5-flash', api_key='via-arg')
-    agent_model = m.agent_model({}, True, None)
+    agent_model = await m.agent_model({}, True, None)
```

Comment:
I will say that it might be nice if these are forced to be keyword arguments, it'd be easy to mess up and do `{}, None, True` or something and with None being falsely and whatnot it might fail in surprising ways.

Reply:
This is basically an internal API, otherwise I would agree.